Library/ViewModels/BackingViewModel.swift

```diff
@@ -263,6 +305,19 @@ public final class BackingViewModel: BackingViewModelType, BackingViewModelInput
   public var outputs: BackingViewModelOutputs { return self }
 }
 
+private func shouldHideMarkReceived(backing: Backing, project: Project) -> Bool {
+
+  if !project.memberData.permissions.isEmpty {
```

Comment:
We are wondering if you're a collaborator or creator and you backed your own project, we wouldn't necessarily want to hide this.